.gitignore

```diff
@@ -62,7 +51,7 @@ jspm_packages
 .LSOverride
 
 # Icon must end with two \r
-Icon
+Icon

 
```

Comment:
I removed from both solution and master in commits a46b450 and 0b858ae, respectively.